austruckie/settings.py

```diff
@@ -22,7 +22,7 @@
 SECRET_KEY = '96m&w0p21+k+x6#&3*@l09_ec*!6aelvzj4)wl*ly8n_j%xw#+'
 
 # SECURITY WARNING: don't run with debug turned on in production!
-DEBUG = False
+DEBUG = True
```

Comment:
Hi Ali, kindly remove settins.py from the commit. DEBUG cant be true in production